### ISSUE TITLE
unable to combine MONDO concept annotations with other types

### ISSUE THREAD

**Issue by krufel** [UCDenver-ccp/CRAFT#21]
Hello, and thanks so much for maintaining this valuable resource!

I have encountered an issue with MONDO annotations that I outline below.

Using the conversion tool, I struggle to generate output containing both MONDO and PR concept annotations (in fact any other type I have tried to combine with MONDO results in the same issue).

For example, using this command: 

`boot part-of-speech  coreference concept -t PR concept -t MONDO convert -r  -o ~/Desktop/craft-coref-brat`

The stacktrace contains the following (shortened):
```
Caused by: java.lang.IllegalArgumentException: exception thrown when parsing the following: (TOP (TITLE (NP (NP (JJ Molecular) (NN cloning)) (CC and) (NP (NP (NN characterization)) (PP (IN of) (NP (DT the) (NN mouse) (NN Acdp) (NN gene) (NN family)))))))

...

Caused by: java.lang.IllegalArgumentException: plain text does not align with tokens in treebank parse.  node text = 'Molecular'  plain text = 'Dorsovent'

```

Similarly, using the more simple command `boot concept -t MONDO concept -t PR convert -r  -o ~/Desktop/craft-coref-brat`

the stacktrace contains (shortened):

```
java.lang.StringIndexOutOfBoundsException: begin 35084, end 35086, length 23624
               clojure.lang.ExceptionInfo: begin 35084, end 35086, length 23624
```

**Comment by bill-baumgartner** [comment]
Hi @krufel,

Thank you very much for reporting this error. It turns out there are two documents in CRAFT that don't have mentions of MONDO concepts, and their Knowtator2 XML files were missing from the distribution. These files have been added and are now available in release `5.0.1`. The boot commands you were previously trying should now work. Please let me know if you run into further issues.

Thanks,

Bill

**Comment by krufel** [comment]
Excellent, can confirm that this works for me now. 

Thanks for looking into this, and once more for maintaining this valuable resource!

Felix
